mlir/test/python/dialects/transform_structured_ext.py

```diff
@@ -355,7 +355,7 @@ def testTileCompact(target):
 @create_sequence
 def testTileAttributes(target):
     attr = DenseI64ArrayAttr.get([4, 8])
-    ichange = DenseI64ArrayAttr.get([0, 1])
+    ichange = Attribute.parse("[0, 1]")
```

Comment:
```suggestion
    ichange = [0, 1]
```

I believe this will work for `attr` as well.

Reply:
But isn't the point of that test (`testTileAttributes()`) to pass those as `Attribute`s? Otherwise it would be the same as `testTileCompact()`, or am I missing something?

Reply:
Ah I didn't see the other test. Anyway I'm not sure what the point of this additional test is (I'm away from my desk so I can't see if this builder is an `ext` builder) but the parse shouldn't be necessary for these attributes because of the registered builders in `ir.py`. Anyway it was just nit type comment.

Reply:
@ftynse Do you have an opinion on the seperate tests? Should we keep one with Attributes, one with lists, or just merge them?

Reply:
I'm of the opinion that tests that exercise code, even if the tests themselves are out of date (in terms of idioms and such), should be kept. Like I said - I just hadn't noticed the overlapping tests.